tests/files.ts

```diff
@@ -1,14 +1,147 @@
-import { failUnauthenticated, test } from "./_utilities";
+import { state } from "./_cache";
+import { failUnauthenticated, test, testOperation } from "./_utilities";
 
 test.before(failUnauthenticated);
+test.serial(
+	"getFileWithoutData",
```

Comment:
Following the "title" structure of other tests, this should be `without data`, resulting in the title `getFiles without data` instead of `getFiles getFileWithoutData`.